### DISCUSSION TITLE
v.121.0.1 BETA

### DISCUSSION THREAD

**Discussion by Alex313031** [Alex313031/Mercury-Win7#3] [General]
- This is a beta of Mercury for Windows 7 based on 121.0.1
 - The patches to enable Win7/8/8.1 support past ESR 115 came kindly from @K4Sum1 > https://github.com/Eclipse-Community/vyvfox
 - Window frame is a little messed up
 - I forgot to set the subsystem version to NT 6.0 (He has it set to Windows 95, which reduces some subsystem featuresets)

<hr /><em>This discussion was created from the release <a href='https://github.com/Alex313031/Mercury-Win7/releases/tag/121beta'>v.121.0.1 BETA</a>.</em>

**Comment by spacedrone404** [comment]
@Alex313031 Why so many versions of the same thing? Just unite with @K4sum1 to produce joint-effort product. 
This is just only my vision of situation, it doesn't pretend to be objective.

**Comment by K4sum1** [comment]
Having multiple forks with different names so they can be run at the same time is pretty nice. However this has been stuck on 115,x and 121 for way too long. I want another fork to point to for stuff like DRM or stuff that I won't do though. So I helped make https://github.com/twilight-software/luminafox to fill that hole.

**Reply by spacedrone404** [reply]
> So I helped make https://github.com/twilight-software/luminafox to fill that hole.
So what is the main difference between r3dFox and LuminaFox?

**Reply by K4sum1** [reply]
Not much. There's no theme currently, and slightly different defaults. It has the weird level 3 or whatever DRM. idk what else
